docs/docs/features/studio.md

```diff
@@ -10,6 +10,37 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | 🕯️ Low Priority | ❌ Not Planned |
+| :------ | :----------- | :------------------- | :--------- | :-------------- | -------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
+|      ✅       | Making changes while connected via BLE (Linux web-app & native apps only)                         |
+|      ✅       | Assigning [predefined behaviors](../keymaps/behaviors/index.mdx) to keys                          |
+|      ✅       | Assigning [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors) to keys |
+|      💡       | [Configuring predefined behaviors](../config/behaviors.md)                                        |
+|      💡       | [Configuring user-defined behaviors](../config/behaviors.md)                                      |
+|      💡       | [Configuring tap-dance](../keymaps/behaviors/tap-dance.mdx)                                       |
+|      💡       | [Configuring macros](../keymaps/behaviors/macros.md)                                              |
+|      ❌       | Defining and configuring behaviors beyond what is specified by devicetree                         |
```

Comment:
This one *may* give the wrong impression, as I read it.. I wonder how we can succinctly capture the "reserved" concept for this without being too verbose.

Reply:
I think I would change the text "configuring user-defined behaviors" to "configuring user-defined and reserved behaviors" when that turns from planned into a reality, and change this point into "Adding additional reserved behaviors beyond [...]".

I don't think it's necessary to make that distinction yet, as reserved behaviors aren't a thing yet. But I see your point, so... "Defining new behaviors not specified in devicetree"?

Reply:
"Defining new behaviors not specified in devicetree" WFM.

Reply:
Done.